include/aml.h

```diff
@@ -186,50 +185,47 @@ struct aml_area {
 
 /**
  * Low-level function for obtaining memory from an area.
- * @param[in] area: A valid area implementing access to the target memory.
- * @param[in] size: The usable size of memory to obtain.
- * @param[in, out] opts: Opaque handle to pass additional options to the area.
+ * @param[in] area: a valid area implementing access to the target memory.
+ * @param[in] size: the usable size of memory to obtain.
+ * @param[in, out] opts: opaque handle to pass additional options to the area.
  * @return a pointer to the memory range of the requested size allocated
- * within the area.
- * @return NULL on failure, with aml_errno set to the appropriate error
- * code.
+ * within the area ; NULL on failure, with aml_errno set to the appropriate
+ * error code.
  **/
-void *aml_area_mmap(const struct aml_area        *area,
-		    size_t                        size,
-		    struct aml_area_mmap_options *opts);
+void *aml_area_mmap(const struct aml_area *area,
+                    size_t size,
+                    struct aml_area_mmap_options *opts);
 
 /**
- * Releases memory region obtained with aml_area_mmap().
- * @param area: A valid area implementing access to the target memory.
- * @param ptr: A pointer to the memory obtained with aml_area_mmap()
+ * Release memory region obtained with aml_area_mmap().
+ * @param[in] area: a valid area implementing access to the target memory.
+ * @param[in, out] ptr: a pointer to the memory obtained with aml_area_mmap()
  *        using the same "area" and "size" parameters.
- * @param size: The size of the memory region pointed to by "ptr".
+ * @param[in] size: the size of the memory region pointed to by "ptr".
  * @return 0 if successful, an error code otherwise.
  * @see aml_area_mmap()
  **/
-int
-aml_area_munmap(const struct aml_area *area,
-		void                  *ptr,
-		size_t                 size);
+int aml_area_munmap(const struct aml_area *area, void *ptr, size_t size);
 
 /**
  * Print on the file handle the metadata associated with this area.
- * @param stream the stream to print on
- * @param prefix prefix to use on all lines
- * @param area area to print
+ * @param[out] stream: the stream to print on
```

Comment:
```suggestion
 * @param[in] stream: the stream to print on
```
Same as the function pointer definition. As a rule, there shouldn't be any difference between the generic API and the implementation prototypes, outside of very specific reasons.